README.md

```diff
@@ -74,41 +75,25 @@ npm install --save @material/react-button
 
 If you want to use the compiled CSS and not customize any colors, text, etc. you can skip to [Step 3a](#step-3a-use-compiled-css).
 
-Most likely you'll want to start using the [Sass mixins](https://github.com/material-components/material-components-web/blob/master/docs/code/architecture.md#sass) to customize your app. There are a few ways to achieve this. `create-react-app` does have a [recommended approach](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-a-css-preprocessor-sass-less-etc), which we also recommend.
-
-The following is an alternate version of the `create-react-app` approach. The difference being all the `node_modules` imports will go into `./src/App.scss`. First install `node-sass-chokidar`:
+Most likely you'll want to start using the [Sass mixins](https://github.com/material-components/material-components-web/blob/master/docs/code/architecture.md#sass) to customize your app. MDC Sass files are not supported out of the box, since we do not append `~` to our module imports. See this [Github issue](https://github.com/facebook/create-react-app/issues/4494#issuecomment-428531848) explaining the issue in detail. There is a workaround, but requires some work on your end (we promise it is not too difficult).
 
-```
-npm install -D node-sass-chokidar npm-run-all
-```
+##### Add environment variable
 
-In `package.json` replace the following:
+To get MDC React Components to work with `create-react-app` you need to set a `SASS_PATH` environment variable that points to your `node_modules` directory. To quickly do this on OSx or Linux enter the following in your command line:
 
-```js
-"scripts": {
-  "start": "react-scripts start",
-  "build": "react-scripts build",
-  "test": "react-scripts test --env=jsdom",
-  "eject": "react-scripts eject"
-}
+```sh
+export SASS_PATH=./node_modules
 ```
 
-with:
+If you're on Windows use the following:
 
-```js
-"scripts": {
-  "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules ./src/App.scss -o ./src",
-  "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules --watch ./src/App.scss ./src/App.css",
-  "start-js": "react-scripts start",
-  "start": "npm-run-all -p watch-css start-js",
-  "build-js": "react-scripts build",
-  "build": "npm-run-all build-css build-js",
-  "test": "react-scripts test --env=jsdom",
-  "eject": "react-scripts eject"
-}
+```bat
+SET SASS_PATH=./node_modules
```

Comment:
`/` is not the file separator on windows. have we tested on windows?

Reply:
yup tested on windows...good catch thanks :)